src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx

```diff
@@ -105,87 +112,94 @@ Rate limits as enforced by Azure will be shared between any current polling inte
-- Review the [integration settings](#integration-settings) section to adjust polling intervals and filters based on your observability requirements.
-- Start with those resource types that were not previously supported by New Relic with traditional Azure integration.
+* Start by validating and testing the new integration on testing environments first.
+* Review the [integration settings](#settings) section to adjust polling intervals and filters based on your observability requirements.
+* If you were a previous user of our older Azure integrations: Start with those resource types that we didn't previously support.
 
+Note that the Azure Monitor integration and other Azure integrations can be enabled in parallel. See the [querying your data](#Querying-Your-Data) section to learn how to query Azure Monitor metrics in isolation.
 
-Note that the Azure Monitor integration and other Azure integration can be enabled in parallel. See the [querying your data](#Querying-Your-Data) section to learn how to query Azure Monitor metrics in isolation
+## Integration settings [#settings]
 
-## Integration settings
+Polling frequency details for the Azure Monitor integration:
 
-Polling frequency:
-- `Metrics polling interval`: how often will New Relic fetch metrics from your resources through Azure Monitor
-- `Metadata and tags polling interval`: how often will New Relic fetch metadata and tags from your Azure resources through Azure Monitor
+* `Metrics polling interval`: how often we fetch metrics from your resources.
+* `Metadata and tags polling interval`: how often we fetch metadata and tags from your resources.
 
 Data collection and filters:
 
-- `Limit to resource types`: only fetch data from resources matching specified resource type(s) (if enabled and not empty)
-- `Limit to resource group`: only fetch data from resources matching specified resource group(s) (if enabled and not empty)
-- `Resource tags to include`: only fetch data from resources tagged with specified tag(s) (if enabled and not empty)
-- `Resouce tags to exclude`: drop data from resources tagged with specified tag(s) (highest precedence)
+* `Limit to resource types`: only fetch data from resources matching specified resource types (if enabled and not empty)
+* `Limit to resource group`: only fetch data from resources matching specified resource groups (if enabled and not empty)
+* `Resource tags to include`: only fetch data from resources tagged with specified tags (if enabled and not empty)
+* `Resouce tags to exclude`: drop data from resources tagged with specified tags (highest precedence)
+
+## Out of the box support [#support]
 
+Here are some details on the features you get with this integration by default, out of the box: 
 
-## Out of the box support
 <CollapserGroup>
   <Collapser
-    className="freq-link"
     id="query"
     title="Querying your data"
   >
 
 Metrics collected using the Azure Monitor:
```

Comment:
I'd love a little more context in each of these bullets for why these things are important.